layers/decompression/decompression.cpp

```diff
@@ -17,7 +17,7 @@
  * Author: Vikram Kushwaha <vkushwaha@nvidia.com>
  */
 
-static bool logging_enabled = false;
+static bool logging_enabled = true;
```

Comment:
I didn't see any logging in the test run output.

Reply:
I didn't see any testing either.